Jenkinsfile

```
@@ -44,7 +44,7 @@ pipeline {
         stage('Build library and publish') {
             steps {
                 script {
-                    nodejs(nodeJSInstallationName: 'nodejs14') {
+                    nodejs(nodeJSInstallationName: 'nodejs16.14.2') {
```

```suggestion
                    nodejs(nodeJSInstallationName: 'nodejs16') {
```

Je préfère qu'on garde cette façon de nommer la version node sans préciser le détail. J'ai modifié côté Jenkins.